src/main/resources/languages/messages_en.yml

```diff
@@ -208,6 +208,8 @@ messages:
   - '&7This one was actually a Biscuit and not a Cookie'
   - '&7Neon signs are LIT!'
 
+  above-limit-level: '&cCannot enchant a enchantment level higher than %level%'
```

Comment:
this is used for enchant and disenchant, little weird. Probably should be separate